src/patternfly/components/DataList/examples/DataList.md

```diff
@@ -296,7 +296,7 @@ When a list item includes more than one block of content, it can be difficult fo
 
 ### Expandable compact
 ```hbs
-{{#> data-list data-list--modifier="pf-m-compact" data-list--id="data-list-expandable" data-list--attribute='aria-label="Expandable data list example"'}}
+{{#> data-list data-list--modifier="pf-m-compact" data-list--id="data-list-expandable-compact" data-list--attribute='aria-label="Expandable data list example"'}}
```

Comment:
"[secondary/tertiary] content details" both show up 4 times in the expandable examples, too. Is this an axe error message? I don't see it when I run it in my browser. I imagine this would be a pretty widespread issue.

Reply:
I think it would be flagged as an issue - but the aria-labels in the expandable examples are being generated incorrectly. For example, the first expandable content in the expandable compact example has an aria-label= `data-list-expandable-compact Primary content details`